config/odp-linux-generic.conf

```diff
@@ -87,6 +87,13 @@ sched_basic: {
 	# value is the number of threads using the scheduler.
 	prio_spread = 4
 
+	# Weight of the preferred scheduler internal queue over other queues.
+	# Each thread prefers one of the scheduler internal queues over others.
+	# This value controls how many times the preferred queue is polled
+	# between polls to other queues. A higher value typically improves
+	# parallelism, but causes uneven service level for low thread counts.
+	prio_spread_weight = 63
```

Comment:
Should have a better explanation of how this number is specified and used. What's the valid range? What are the different behaviors at the extreme ends of that range? Where does this default value sit in that range?

Reply:
This is what is does: "...how many times the preferred queue is polled between polls to other queues". For example, poll the preferred queue 63 times, poll another queue one time, poll the preferred queue 63 times, poll another queue one time, ...

Since this is not API, user is expected to have a bit of knowledge on the implementation (code) before changing the values ... or just find optimal value through trial and error.

I have not copied range into config file documentation since it would become outdated easily. The code have range checks and prints an error (with min/max values) when range is exceeded.

Reply:
Updated text for both prio_spread configs on v2. Hope, it's better now.

Reply:
A config file should contain sufficient documentation so that an application developer can use it to tune their app without having to read the source code to try to figure that out. If ODP is installed on a system what most developers will have is the binaries, the user guide, and this config file. Those should be sufficient to be able to use ODP.